### DISCUSSION TITLE
Applying site template with New-PnPTenantSite fails with "Error applying default column values".

### DISCUSSION THREAD

**Discussion by wataruf01** [pnp/powershell#2819] [Q&A] [answered]
PnP.PowerShell to use the site template.
Get-PnPSiteTemplate ・・・ Get the site template
Invoke-PnPSiteTemplate ・・・ Apply site template

When applying a template with Invoke-PnPSiteTemplate, the message "Error applying default column values" may appear and fail.

If anyone has information on how this error occurs, please let me know.

The following figure shows the results of my runs when I tried each version.
I am not sure if the version is the cause of the error .

![スクリーンショット 2023-02-14 201615](https://user-images.githubusercontent.com/39029146/218721392-10b638e6-f5a2-4c6c-8a90-e08dff059195.png)

**Comment by AndersRask** [comment, marked as answer]
Whenever you get errors invoking a site template, try and enable tracing. This would probably give you relevant info about what is missing on the site.

`Set-PnPTraceLog -On -LogFile traceoutput.txt`

Regarding the specific error with default values, not fields supports this. 
This leads to the question: Is the template code exported from an existing site, or made "by hand"?
A few tips to debug template issues could be:
Does the template work on another site?
If so, what are the differences?
Try simplifying the template code to pinpoint exactly where the error occurs

**Comment by wataruf01** [comment]
Sorry for leaving the thread unattended.

The matter has since been resolved.

I unblocked "security blocking" on all of the files used as PowerShell input information. I did so and was able to avoid the error.

I have marked as answered AndersRask's reply that provided the solution tip.

Thank you very much for your advice.
